packages/server/server.js

```diff
@@ -94,6 +94,10 @@ const runServer = (
     res.status(200).send({ ip: req.ip })
   })
 
+  app.get('/atlantic-cells.json', (req, res) => {
```

Comment:
This requires us to use this particular file-name.  They may need to have a number of different files of data.

I appreciate we need to define some kind of route.  How about if the request begins with `/cells/` - and we provide the contents of the file at `/cells/xxxxx`?

Reply:
would be great, good idea 💡